environment.go

```diff
@@ -406,6 +394,20 @@ func (e *envState) createRemoteQueues(ctx context.Context) error {
 	return catcher.Resolve()
 }
 
+func dbCreds(ctx context.Context) (options.Credential, error) {
+	awsCreds, err := ec2rolecreds.New().Retrieve(ctx)
+	if err != nil {
+		return options.Credential{}, errors.Wrap(err, "getting AWS credentials")
+	}
+	return options.Credential{
+		AuthMechanism:           "MONGODB-AWS",
+		AuthMechanismProperties: map[string]string{"AWS_SESSION_TOKEN": awsCreds.SessionToken},
```

Comment:
nit: Can we please use a const for AWSSessionToken and MONGODB-AWS?  (We have AWSSessionToken as a const in assume_ec2_role.go)

Reply:
done